packages/cli/src/migrations/deleteOldPeerstore.ts

```diff
@@ -0,0 +1,31 @@
+import {rename} from "node:fs/promises";
+import {LevelDbController} from "@chainsafe/lodestar-db";
+import {ILogger} from "@chainsafe/lodestar-utils";
+
+/**
+ * As of libp2p v0.36.0 (https://github.com/libp2p/js-libp2p/commit/978eb3676fad5d5d50ddb28d1a7868f448cbb20b)
+ * the peerstore format has changed in a breaking way.
+ *
+ * Because of that, we need to wipe the old peerstore if it exists.
+ */
+export async function deleteOldPeerstore(peerStoreDir: string, logger: ILogger): Promise<void> {
+  const db = new LevelDbController({name: peerStoreDir}, {logger});
+  await db.start();
+
+  const keys = await db.keys();
```

Comment:
The issue with this approach is that we need to read the entire db always. What if:
- We add a new key value pair with metadata information like the version of the libp2p consumer of this peerstore
- If this key is not found, delete db + write key after wipe
- If key is found use for future migrations

Reply:
This DB isn't managed by lodestar. If you do do that, there will be a fatal error, see @tuyennhv 's error above
The peerstore doesn't currently allow any stray keys in the database, nor should we (lodestar) be using this DB for that purpose.

Reply:
Oh see, how many keys are there in a typical peer store? We should do a rough benchmark of the performance cost of reading all of them at start-up, at least to be aware